platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -287,6 +287,16 @@ public void run() {
                 markers.clear();
             }
         });
+
+        // set all mapController listener references to null
```

Comment:
I think it's pretty clear _what_ this does, but it will be useful to explain _why_ we do it.

Reply:
Sure will expand on that.